owners/OWNERS.example.yaml

```diff
@@ -58,17 +59,40 @@
 - "*.{js,css}": frontend
 - "package.{,lock.}json": somebodyelse
 
+# Prepending a user or team name with a `?` will prevent this owner from being
```

Comment:
Line 25: Curious: Does the list have to be indented? Or is that just good for readability?

Reply:
YAML is extremely whitespace-sensitive :grimacing:

Reply:
In that case, probably worth mentioning explicitly in the comment.

Reply:
Should I leave a link to a guide to YAML in general? There's nothing special about the treatment of it here